code/modules/mining/satchel_ore_boxdm.dm

```diff
@@ -31,25 +31,28 @@
 	var/amt_plasma = 0
 	var/amt_uranium = 0
 	var/amt_clown = 0
+	var/amt_mime = 0
 	var/amt_bluespace = 0
 
 	for (var/obj/item/weapon/ore/C in contents)
 		if (istype(C,/obj/item/weapon/ore/diamond))
-			amt_diamond++;
+			amt_diamond++
 		if (istype(C,/obj/item/weapon/ore/glass))
-			amt_glass++;
+			amt_glass++
 		if (istype(C,/obj/item/weapon/ore/plasma))
-			amt_plasma++;
+			amt_plasma++
 		if (istype(C,/obj/item/weapon/ore/iron))
-			amt_iron++;
+			amt_iron++
 		if (istype(C,/obj/item/weapon/ore/silver))
-			amt_silver++;
+			amt_silver++
 		if (istype(C,/obj/item/weapon/ore/gold))
-			amt_gold++;
+			amt_gold++
 		if (istype(C,/obj/item/weapon/ore/uranium))
-			amt_uranium++;
```

Comment:
No seriously, who thought that these were needed? Did they transition directly from another programming language without bothering to learn DM syntax?